{"name": "Проверка валидности поля номера телефона", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 10s, while waiting for:\nbrowser.element(('xpath', \"//div[contains(text(),'некорректный номер телефона')]\")).is visible\n\nReason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'некорректный номер телефона')]\"}\n  (Session info: chrome=113.0.5672.94)\n\nScreenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320339655\\1684320339660.png\nPageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320339655\\1684320339660.html", "trace": "venv\\lib\\site-packages\\selene\\core\\wait.py:102: in for_\n    return fn(self._entity)\nvenv\\lib\\site-packages\\selene\\core\\condition.py:136: in __call__\n    return self._fn(*args, **kwargs)\nvenv\\lib\\site-packages\\selene\\core\\condition.py:91: in fn\n    if not predicate(entity):\nvenv\\lib\\site-packages\\selene\\core\\match.py:37: in <lambda>\n    'is visible', lambda element: element().is_displayed()\nvenv\\lib\\site-packages\\selene\\core\\entity.py:179: in __call__\n    return self._locator()\nvenv\\lib\\site-packages\\selene\\core\\locator.py:34: in __call__\n    return self._locate()\nvenv\\lib\\site-packages\\selene\\core\\entity.py:1529: in <lambda>\n    f'{self}.element({by})', lambda: self.driver.find_element(*by)\nvenv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:976: in find_element\n    return self.execute(Command.FIND_ELEMENT, {\nvenv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:321: in execute\n    self.error_handler.check_response(response)\nvenv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py:242: in check_response\n    raise exception_class(message, screen, stacktrace)\nE   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'некорректный номер телефона')]\"}\nE     (Session info: chrome=113.0.5672.94)\n\nDuring handling of the above exception, another exception occurred:\ntests\\test_login.py:90: in test_field_phone\n    page.registration_field_phone_number()\npages\\home.py:182: in registration_field_phone_number\n    self.wait_element(self.message_error_phone_text_var_string)\npages\\base.py:57: in wait_element\n    browser.s(locator).should(be.visible)\nvenv\\lib\\site-packages\\selene\\core\\entity.py:540: in should\n    super().should(condition)\nvenv\\lib\\site-packages\\selene\\core\\entity.py:127: in should\n    self.wait.for_(condition)\nvenv\\lib\\site-packages\\selene\\core\\wait.py:125: in for_\n    raise self._hook_failure(failure)\nE   selene.core.exceptions.TimeoutException: Message: \nE   \nE   Timed out after 10s, while waiting for:\nE   browser.element(('xpath', \"//div[contains(text(),'некорректный номер телефона')]\")).is visible\nE   \nE   Reason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'некорректный номер телефона')]\"}\nE     (Session info: chrome=113.0.5672.94)\nE   \nE   Screenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320339655\\1684320339660.png\nE   PageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320339655\\1684320339660.html"}, "steps": [{"name": "Нажать Аккаунт", "status": "passed", "start": 1684320784644, "stop": 1684320784691}, {"name": "Нажать Зарегистрироваться", "status": "passed", "start": 1684320784691, "stop": 1684320784849}, {"name": "Валидация поля Номер телефона", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 10s, while waiting for:\nbrowser.element(('xpath', \"//div[contains(text(),'некорректный номер телефона')]\")).is visible\n\nReason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'некорректный номер телефона')]\"}\n  (Session info: chrome=113.0.5672.94)\n\nScreenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320339655\\1684320339660.png\nPageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320339655\\1684320339660.html\n\n", "trace": "  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\pages\\home.py\", line 182, in registration_field_phone_number\n    self.wait_element(self.message_error_phone_text_var_string)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\pages\\base.py\", line 57, in wait_element\n    browser.s(locator).should(be.visible)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\entity.py\", line 540, in should\n    super().should(condition)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\entity.py\", line 127, in should\n    self.wait.for_(condition)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\wait.py\", line 125, in for_\n    raise self._hook_failure(failure)\n"}, "steps": [{"name": "Заполнение поля '' текстом '999888776'", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000002BB4ED2C250>"}, {"name": "text", "value": "999888776"}, {"name": "fieldName", "value": "''"}], "start": 1684320784849, "stop": 1684320784968}, {"name": "Взять текст из элемента ''Текст ошибки поля  пароль не верный или текст отсутствует''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000002BB4ED2CFD0>"}, {"name": "allureText", "value": "'Текст ошибки поля  пароль не верный или текст отсутствует'"}], "start": 1684320792017, "stop": 1684320792054}, {"name": "Заполнение поля '' текстом '999888775'", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000002BB4ED2C250>"}, {"name": "text", "value": "999888775"}, {"name": "fieldName", "value": "''"}], "start": 1684320794056, "stop": 1684320794141}, {"name": "Взять текст из элемента ''Текст ошибки поля  пароль не верный или текст отсутствует''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000002BB4ED2CFD0>"}, {"name": "allureText", "value": "'Текст ошибки поля  пароль не верный или текст отсутствует'"}], "start": 1684320801181, "stop": 1684320801196}, {"name": "Заполнение поля '' текстом ''9998887m''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000002BB4ED2C250>"}, {"name": "text", "value": "'9998887m'"}, {"name": "fieldName", "value": "''"}], "start": 1684320803203, "stop": 1684320803275}, {"name": "Взять текст из элемента ''Текст ошибки поля  пароль не верный или текст отсутствует''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000002BB4ED2CFD0>"}, {"name": "allureText", "value": "'Текст ошибки поля  пароль не верный или текст отсутствует'"}], "start": 1684320810326, "stop": 1684320810343}, {"name": "Заполнение поля '' текстом '9998887766'", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000002BB4ED2C250>"}, {"name": "text", "value": "9998887766"}, {"name": "fieldName", "value": "''"}], "start": 1684320812348, "stop": 1684320812434}, {"name": "Проверка наличия элемента", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 10s, while waiting for:\nbrowser.element(('xpath', \"//div[contains(text(),'некорректный номер телефона')]\")).is visible\n\nReason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'некорректный номер телефона')]\"}\n  (Session info: chrome=113.0.5672.94)\n\nScreenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320339655\\1684320339660.png\nPageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320339655\\1684320339660.html\n\n", "trace": "  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\pages\\base.py\", line 57, in wait_element\n    browser.s(locator).should(be.visible)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\entity.py\", line 540, in should\n    super().should(condition)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\entity.py\", line 127, in should\n    self.wait.for_(condition)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\wait.py\", line 125, in for_\n    raise self._hook_failure(failure)\n"}, "parameters": [{"name": "locator", "value": "'//div[contains(text(),'некорректный номер телефона')]'"}], "start": 1684320819479, "stop": 1684320829940}], "start": 1684320784849, "stop": 1684320829941}], "attachments": [{"name": "Скринтшот", "source": "082b289c-1f3f-4c98-bdb7-d0b407379101-attachment.png", "type": "image/png"}, {"name": "Логи консоли", "source": "33e308f2-6058-45cc-9f0b-54b77d3fc451-attachment.json", "type": "application/json"}, {"name": "log", "source": "3ff6d628-a7c3-4baf-a0b3-918050f92c26-attachment.txt", "type": "text/plain"}], "start": 1684320784644, "stop": 1684320829941, "uuid": "8c285458-ab58-4027-960b-2ea067c7e0e4", "historyId": "98185df0bbce6dad3aecda5b36f83fa6", "testCaseId": "bbd874f4602a7fdc5ab9eaea7165381d", "fullName": "tests.test_login.TestLogin#test_field_phone", "labels": [{"name": "feature", "value": "Тесты логина"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-VLT8PU4"}, {"name": "thread", "value": "12164-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}